### DISCUSSION TITLE
Disable temporarily

### DISCUSSION THREAD

**Discussion by governos** [cssnr/open-links-in-new-tab#36] [Feature Requests]
Hi. It would be neat to have a disable temporarily function as well.
Simply toggled with the `Enable temporarily` button and keybind.
Cheers.

**Comment by smashedr** [comment]
Psychic... I am actually working on this just today in this PR: https://github.com/cssnr/open-links-in-new-tab/pull/35

The new version will allow Enabling and Disabling site via a Toggle Keybind, Context Menu, or Popup Click.

I also plan changing Enable Temporary to Toggle Temporary, and will also have a Toggle Keybind, Context Menu, or Popup Click to Enable/Disable the current tab only.

Lastly, It will have a new feature, Enable Globally, that can be toggled on/off via a Toggle Keybind, Context Menu, or Popup Click to Enable/Disable the addon globally until toggled off.

And Hopefully be able to update shadowRoots.

Thanks for the feedback, I will keep you posted as we get closer to a release. Let me know if you have any more questions.

**Comment by governos** [comment]
That's great news, thank you.
